tests/test_middleware_django.py

```
@@ -0,0 +1,235 @@
+
+class TestScalekitAuthDjangoConstruction(unittest.TestCase):
+    def tearDown(self):
+        get_config.cache_clear()
+
+    def test_missing_cookie_secret_raises_immediately(self):
+        get_config.cache_clear()
+        django_settings.SCALEKIT_COOKIE_ENCRYPTION_SECRET = ""
+        django_settings.SCALEKIT_CLIENT = MagicMock()
+        with self.assertRaises(ValueError):
+            get_config()
+        django_settings.SCALEKIT_COOKIE_ENCRYPTION_SECRET = "django-test-cookie-secret"
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to: "SCALEKIT_COOKIE_ENCRYPTION_SECRET"

(S105)